examples/plot_knockoff_aggregation.py

```diff
@@ -124,7 +123,7 @@ def single_run(
 
 for seed in seed_list:
```

Comment:
This loop is embarrassingly parallel, we could try using joblib.

Reply:
We need to choose if we do the boot strapping in parallel or if we do this loop in parallel. 
Using the 2 won't be optimal.